FiftyOne.Pipeline.Web.sln

```diff
@@ -67,170 +67,180 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "AspNet Framework MVC Exampl
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "AspNetCore Example", "Web Integration\Examples\NetCore\AspNetCore Example.csproj", "{AF8687D1-E12A-430F-A616-FBB0EB5D92D6}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "FiftyOne.Pipeline.Web.Framework.Tests", "Web Integration\Tests\FiftyOne.Pipeline.Web.Framework.Tests\FiftyOne.Pipeline.Web.Framework.Tests.csproj", "{4E0D24CA-D682-4BFC-B506-31B9BE0DEA31}"
```

Comment:
I suspect this will cause a problem with `dotnet build` as there's a Framework project in there.

Reply:
The solution contains both Core and Framework so this new test should be included.